revscoring/datasources/tests/test_page_creation.py

```diff
@@ -1,28 +1 @@
-from deltas import Delete, Equal, Insert
-from mw import Timestamp
-from nose.tools import eq_
-
 from .. import page_creation
```

Comment:
My thought is that, even if there is nothing to test within the module, this test will still make sure we can import it.  Same to the others below.